iac/terraform/aws/web-client/route53.tf

```diff
@@ -0,0 +1,22 @@
+locals {
```

Comment:
if we have separate file local variables, let's put this there

Reply:
Moved